spec/commands/get_fee_balance_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe GetFeeBalance do
+  def perform
+    described_class.for
+  end
+
+  let(:balance) { -145000 }
+  let(:ledger_account) { create(:user_ledger_account, balance: balance) }
+
+  before do
+    ENV["BUSINESS_USER_ID"] = ledger_account.accountable_id.to_s
```

Comment:
Esto puede hacer un `stub` de la siguiente forma:

`allow(ENV).to receive(:fetch).with('BUSINESS_USER_ID').and_return(ledger_account.accountable_id.to_s)`

Reply:
buenísimo, gracias!